src/Auth/Directives.js

```diff
@@ -108,11 +108,34 @@ class AuthenticatedDirective extends SchemaDirectiveVisitor {
   }
 }
 
+class OwnerDirective extends SchemaDirectiveVisitor {
 
+  visitObject(type){
```

Comment:
This visitObject override can be removed as it is not used by the rest of your code and the wrapOwnerAuth function that it calls does not exist (or was no part of this pull request).